calico_rkt/calico_rkt.py

```diff
@@ -167,32 +175,30 @@ def _container_remove(hostname, orchestrator_id, container_id, client):
     netns.remove_veth(endpoint.name)
 
     # Remove the container from the datastore.
-    client.remove_workload(hostname, orchestrator_id, container_id)
+    client.remove_workload(hostname=hostname, 
+                           orchestrator_id=orchestrator_id, 
+                           workload_id=container_id)
 
-    print_stderr("Removed Calico interface from %s" % container_id)
+    logger.info("Removed Calico interface from %s" % container_id)
 
 def _create_profile(endpoint, profile_name, ip, client):
```

Comment:
This function primarily sets the profile of an endpoint, and optionally creates it as well. The function name should probably be `set_endpoint_profile`